Model/Sync/Checkout/Data.php

```diff
@@ -154,6 +161,14 @@ public function prepareData(Quote $quote)
 
         $billingAddressData = $this->prepareAddress($quote, true);
         if (!$billingAddressData || !$billingAddressData['country_code']) {
+            $this->checkoutLogger->info(
+                __(
+                    'Failed to sync Checkout to Yotpo - Billing Address didnt contain valid Country ID - Checkout ID: %1, Country ID: %2',
```

Comment:
Billing Address **didn't** contain valid Country
(You can use "did not" if you feel like).